index.md

```diff
@@ -6,5 +6,16 @@ lang-ref: covid19-notification-app-website-index
 ---
 
 <div class="md-block-lead" markdown="1">
-We krijgen het coronavirus steeds meer onder controle, maar het is er nog altijd. Dus is het belangrijk om alert te blijven. Daarom is er een app in ontwikkeling. 
+ {{ site.data.translations.info-lead[page.lang] }}
+  <h2>
+    {{ site.data.translations.info-lead-title[page.lang] }}
+  </h2>
+  <div class="lead__app_badges">
+    <a class="app_badge_column__left" href="{{ site.data.translations.info-app-store-url[page.lang] }}">
+      <img src="{{ relroot }}img/{{ site.data.translations.info-app-store-img[page.lang] }}" alt="App Store" />
```

Comment:
Deze images zijn de enige content van de link. Dat betekent dat hun `alt` tekst de linktekst wordt. Dit kan het best een actie zijn: 

```suggestion
       <img src="{{ relroot }}img/{{ site.data.translations.info-app-store-img[page.lang] }}" alt="Download in de App Store" /> 
```

En omdat ie in meerdere talen bestaat, zou dat denk ik een vertaalstring moeten worden.